index.js

```diff
@@ -4,15 +4,49 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+updateTitle = header.style.color = "Green"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+let travelerProfile = document.querySelector(".traveler")
+let imgUrl = travelerProfile.querySelector("img")
+imgUrl.src = traveler.photo
+imgUrl.alt = traveler.name
+
+let nameUpdate = travelerProfile.querySelector("h2")
+nameUpdate.textContent = traveler.name
+
+let nicknameUpdate = travelerProfile.querySelector("em")
+nicknameUpdate.textContent = traveler.nickname
```

Comment:
Nice job scoping your search! If you want to scope it even more, you could do `let travelerProfile = document.querySelector("#profile")`.